### PR TITLE
ticdc: remove pulsar

### PR DESCRIPTION
**Pull request by hi-rustin**
Signed-off-by: hi-rustin <rustin.liu@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

remove pulsar, we removed it from the code.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by shichun-0415** [comment]
@hi-rustin @leoppro Maybe we should also mention the removal of the experimental feature in [6.3 release notes](https://github.com/pingcap/docs-cn/pull/11115).

**Comment by hi-rustin** [comment]
> @hi-rustin @leoppro Maybe we should also mention the removal of the experimental feature in [6.3 release notes](https://github.com/pingcap/docs-cn/pull/11115).

Yes. I think we need to add it.

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -140,7 +140,7 @@ Info: {"sink-uri":"mysql://root:123456@127.0.0.1:3306/","opts":{},"create-time":
 ```
 
 - `--changefeed-id`：同步任务的 ID，格式需要符合正则表达式 `^[a-zA-Z0-9]+(\-[a-zA-Z0-9]+)*$`。如果不指定该 ID，TiCDC 会自动生成一个 UUID（version 4 格式）作为 ID。
-- `--sink-uri`：同步任务下游的地址，需要按照以下格式进行配置，目前 scheme 支持 `mysql`/`tidb`/`kafka`/`pulsar`。
+- `--sink-uri`：同步任务下游的地址，需要按照以下格式进行配置，目前 scheme 支持 `mysql`/`tidb`/`kafka`。
```
Comment (shichun-0415) [inline ticdc/manage-ticdc.md:143]:
```suggestion
- `--sink-uri`：同步任务下游的地址，需要按照以下格式进行配置，目前 scheme 支持 `mysql`、`tidb` 和 `kafka`。
```

File: ticdc/ticdc-faq.md [hunk 2]
```diff
@@ -119,7 +119,7 @@ cdc cli changefeed create --pd=http://10.0.10.25:2379 --sink-uri="kafka://127.0.
 > **注意：**
 >
 > * 该功能在 TiCDC 4.0.2 版本引入。
-> * 目前 TiCDC 仅支持将 Canal 格式的变更数据输出到 MQ 类的 Sink（例如：Kafka，Pulsar）。
+> * 目前 TiCDC 仅支持将 Canal 格式的变更数据输出到 MQ 类的 Sink（例如：Kafka）。
```
Comment (shichun-0415) [inline ticdc/ticdc-faq.md:122]:
```suggestion
> * 目前 TiCDC 仅支持将 Canal 格式的变更数据输出到 MQ 类的 Sink（例如 Kafka）。
```

**Review by shichun-0415 — APPROVED** [review]
Rest LGTM

**Review by leoppro — APPROVED** [review]

File: ticdc/manage-ticdc.md [hunk 3]
```diff
@@ -140,7 +140,7 @@ Info: {"sink-uri":"mysql://root:123456@127.0.0.1:3306/","opts":{},"create-time":
 ```
 
 - `--changefeed-id`：同步任务的 ID，格式需要符合正则表达式 `^[a-zA-Z0-9]+(\-[a-zA-Z0-9]+)*$`。如果不指定该 ID，TiCDC 会自动生成一个 UUID（version 4 格式）作为 ID。
-- `--sink-uri`：同步任务下游的地址，需要按照以下格式进行配置，目前 scheme 支持 `mysql`/`tidb`/`kafka`/`pulsar`。
+- `--sink-uri`：同步任务下游的地址，需要按照以下格式进行配置，目前 scheme 支持 `mysql`、`tidb` 和 `kafka`。
```
Comment (Oreoxmt) [inline ticdc/manage-ticdc.md:143]: [ecosystem-tool-user-guide.md L132](https://github.com/hi-rustin/docs-cn/blob/rustin-patch-pulsar/ecosystem-tool-user-guide.md?plain=1#L132) TiCDC 的输出要不要删除 Apache Pulsar

```markdown
- TiCDC 的输出：TiDB 集群、MySQL、Kafka、Apache Pulsar、Confluent
```

[数据迁移工具概览 L20](https://github.com/hi-rustin/docs-cn/blob/rustin-patch-pulsar/migration-tools.md?plain=1#L20) TiCDC 下游中的 Apache Pulsar 要不要删除

```markdown
| [TiCDC](/ticdc/ticdc-overview.md)| 通过拉取 TiKV 变更日志实现的 TiDB 增量数据同步工具，具有将数据还原到与上游任意 TSO 一致状态的能力，支持其他系统订阅数据变更。|TiDB | TiDB，MySQL，Apache Pulsar，Kafka，Confluent|提供开放数据协议 (TiCDC Open Protocol)。 | TiCDC 只能同步至少存在一个有效索引的表。暂不支持以下场景：暂不支持单独使用 RawKV 的 TiKV 集群。暂不支持在 TiDB 中创建 SEQUENCE 的 DDL 操作和 SEQUENCE 函数。|
```

Comment (hi-rustin) [reply ticdc/manage-ticdc.md:143]: 删除了 谢谢

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Review by shichun-0415 — APPROVED** [review]

**Pull request merged**